Program.cs

```diff
@@ -1,10 +1,40 @@
-﻿namespace GatesApp
+﻿using GatesApp.Raport;
+using GatesApp.RaportItem;
+using GatesApp.Respositories;
+
+namespace GatesApp
 {
     internal class Program
```

Comment:
Del pamokose minetu priezasciu, labai svarbu visada nepamirsti is internal pakeisti i public arba private.